### PR TITLE
Feature/test accessibility

### PR DESCRIPTION
**Pull request by MaryBruff**
What I did:

⭐ Passing accessibility Wave Tool Audit 

Screenshot 2024-01-06 at 3 05 28 PM

Screenshot 2024-01-06 at 3 05 52 PM

⭐ Add more names and words for our search

⚠️ Alerts are for Wiki elements that we cannot change 

Did this break anything?

- [X] No
- [ ]  Yes

Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [ ]  Styling 
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [X]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:

- [ ]  If this code needs to be tested, all tests are passing
- [X]  The code runs locally
- [X]  There are comments where clarification/ organization is needed
- [X]  The code is DRY. If it's not, I tried my best
- [X]  I reviewed my code before pushing

What's next?

⚠️ Finishing PropTypes and Cypress

### CODE REVIEW

File: src/App/App.css [hunk 1]
```diff
@@ -13,15 +13,15 @@ section {
 }
 
 button {
-  background-color: grey;
-  color: white;
+  background-color: #235789;
```
Comment (Jesuitman) [inline src/App/App.css:16]: good thinking doing the contrast for the buttons!

File: src/App/App.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ function App() {
   const showRandomControversy = location.pathname !== '/profile';
   const [savedControversies, setSavedControversies] = useState([]);
 
-  const randomSearchInputs = ['4chan', 'Titan (Submersible)', 'Billy Mitchell (gamer)', 'You Showed Me', 'Ezra Miller', 'Russell Brand', 'Bernie Madoff', 'Amy Winehouse']
+  const randomSearchInputs = ['4chan', 'Titan (Submersible)', 'Billy Mitchell (gamer)', 'You Showed Me', 'Ezra Miller', 'Russell Brand', 'Bernie Madoff', 'Amy Winehouse','Erika Jayne', 'Clinton body count conspiracy theory'];
```
Comment (Jesuitman) [inline src/App/App.js:18]: i am so excited to read these new ones

File: src/hooks/useSearchResults.js [hunk 3]
```diff
@@ -7,93 +7,122 @@ const useSearchResults = () => {
...
   // ... (Logic for displaySearchResults)
   async function displaySearchResults(searchTerm) {
-    const url = 'https://en.wikipedia.org/w/api.php?';
+    const url = "https://en.wikipedia.org/w/api.php?";
     let params = {
-      action: 'query',
-      list: 'search',
-      format: 'json',
-      origin: '*',
+      action: "query",
```
Comment (Jesuitman) [inline src/hooks/useSearchResults.js:12]: did this happen automatically through a tool you used or did you do it manually? cant wait to hear input on that!

**Review by Jesuitman — APPROVED** [review]

File: src/hooks/useSearchResults.js (hunk 3 above)
Comment (JoeyStewart) [reply src/hooks/useSearchResults.js:12]: Also curious about this

**Review by JoeyStewart — APPROVED** [review]

**Pull request merged**